python-phoenixdb/phoenixdb/avatica/client.py

```diff
@@ -148,6 +148,7 @@ def __init__(self, url, max_retries=None, auth=None, verify=None):
         self.max_retries = max_retries if max_retries is not None else 3
         self.auth = auth
         self.verify = verify
+        self.headers = headers or {}
```

Comment:
It would make more sense to merge the {'content-type': 'application/x-google-protobuf'} header and extra headers here in the constructor, and just remove the header parameter from post_request.
We only ever use one set of headers.

Reply:
Done